index.rst

```diff
@@ -1,3 +1,11 @@
+##############################
+LATISS Filter Change Procedure
+##############################
+
+.. abstract::
+
+   This document explains the procedure of what to do when a new filter is added (or going to be added) to the system. Several steps are required in order to ensure ingestion by the Butler such that the data can be served.
```

Comment:
Please, remember to use semantic line breaking, e.g.; split the 2 phrases in 2 separate lines.

Also, this abstract reads a bit funny. I suggest you rephrase it to something like this:

> This document explains the required procedures when replacing or adding a *new*, previously uninstalled, filter or grating to LATISS.
> These steps are required before any data is taken, in order to guarantee the data is correctly handled by the acquisition and ingestion system.